include/ensmallen_bits/ada_delta/ada_delta.hpp

```diff
@@ -64,6 +64,9 @@ class AdaDelta
    * @param tolerance Maximum absolute tolerance to terminate algorithm.
    * @param shuffle If true, the function order is shuffled; otherwise, each
    *        function is visited in linear order.
+   * @param exactObjective Flag that determines whether actual objective over
```

Comment:
Sounds a little bit vague to me, what about "Calculate the exact objective (Default: estimate the final objective obtained on the last pass over the data)."

Reply:
Yes, this is quite clear.